arch/xtensa/src/esp32s3/esp32s3_lcd.c

```diff
@@ -623,6 +663,8 @@ static int IRAM_ATTR lcd_interrupt(int irq, void *context, void *arg)
       regval  = esp32s3_lcd_getreg(LCD_CAM_LCD_USER_REG);
       regval |= LCD_CAM_LCD_START_M;
       esp32s3_lcd_putreg(LCD_CAM_LCD_USER_REG, regval);
+
+      fb_remove_paninfo(&g_base_vtable, FB_NO_OVERLAY);
```

Comment:
Add a return value check.

Reply:
Take a look what I did in my latest changes. It's similar to the other users of this function.